main.py

```diff
@@ -1,10 +1,54 @@
 import init_django_orm  # noqa: F401
-
+import json
+from django.utils import timezone
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_data = json.load(file)
+
+    for player_name, player_data in players_data.items():
+        print(player_data)
+
+        race_name = player_data["race"]["name"]
+        race_description = player_data["race"].get("description", " ")
+
+        race, created = Race.objects.get_or_create(
+            name=race_name,
+            defaults={"description": race_description}
+        )
+
+        for skill_data in player_data["race"].get("skills", []):
+            skill_name = skill_data["name"]
+            skill_bonus = skill_data["bonus"]
```

Comment:
There is no need to create these variables, pass data directly to `get_or_create` here and in the other places